packages/app/package.json

```diff
@@ -40,7 +41,7 @@
     "event-target-shim": "6.0.2",
     "http-server": "0.12.3",
     "jsqr": "1.4.0",
-    "lit": "*",
+    "lit": "2.0.0-rc.2",
```

Comment:
This forces the version (better), without breaking (yay). Previously I had attempted something more recent, which broke.